pkg/scheduler/internal/queue/events.go

```diff
@@ -89,6 +96,44 @@ var (
 	UnschedulableTimeout = framework.ClusterEvent{Resource: framework.WildCard, ActionType: framework.All, Label: "UnschedulableTimeout"}
 )
 
+// PodSchedulingPropertiesChange interprets the update of a pod and returns corresponding UpdatePodXYZ event(s).
+// Once we have other pod update events, we should update here as well.
+func PodSchedulingPropertiesChange(newPod *v1.Pod, oldPod *v1.Pod) (events []framework.ClusterEvent) {
+	podChangeExtracters := []podChangeExtractor{
+		extractPodLabelsChange,
+		extractPodResourceRequestChange,
+	}
+
+	for _, fn := range podChangeExtracters {
+		if event := fn(newPod, oldPod); event != nil {
+			events = append(events, *event)
+		}
+	}
+
+	if len(events) == 0 {
```

Comment:
No, it's not a breaking change because Pod smaller events `Match()` with Pod/Update event, meaning, when plugins have Pod/Update event, they will still receive all events of Pod/update, including smaller events.